prbench-models/src/prbench_models/dynamic3d/ground/parameterized_skills.py

```diff
@@ -1080,7 +1111,9 @@ def sample_parameters(self, x: ObjectCentricState, rng: np.random.Generator) ->
                     np.linalg.norm(
                         np.array(
                             [
-                                pose_x_offset + cupboard_pose.x,
+                                pose_x_offset
+                                + cupboard_pose.x
+                                - 0.03,  # the offset of the cupboard from the cubes.
```

Comment:
we really should put these magic numbers somewhere else (so that it is more clear, and so that it's easier to change in one place later)